README.md

```diff
@@ -37,7 +37,21 @@ If you still cannot get the tool to work, join our [Discord Server](https://disc
 
 ### Running The Ship of Harkinian
 
-Launch the game. If the window immediately closes, or if there are visual artifacts, you may have selected the wrong rom in the OTRGui tool. 
+Launch the game. If the window immediately closes, or if there are visual artifacts, you may have selected the wrong rom in the OTRGui tool.
+
+The default keyboard controls are as follows:
+
+| N64 | A | B | Z | Start | Analog stick | C buttons | D-Pad |
+| - | - | - | - | - | - | - | - |
+| Keyboard | X | C | Z | Space | WASD | Arrow keys | TFGH |
+
+There are also some keyboard shortcuts:
+
+| Keys | Action |
+| - | - |
+| F1 | Show/hide taskbar |
+| F10 | Fullscreen (OpenGL) |
+| Alt+Enter | Fullscreen (DirectX) |
```

Comment:
These fullscreen hotkeys are not really SoH specific. They're OS specific. Nearly any program you can press alt+enter to enter or leave fullscreen. My point is that the alt+enter hotkey is almost always unlisted. Nothing necessarily wrong with listing it here.

Reply:
I don't think any key combo for fullscreen is a universal expectation. There is way too much variation in ways to toggle fullscreen between applications to expect users to know to try any given key/combo

Reply:
I'm mainly including these keys because it's asked incredibly often in the Discord